assets/js/main.js

```diff
@@ -1,23 +1,24 @@
+const apiKey = "9f120f0e95e5cedf05a78abda9ef8477";
+
 async function getMovies() {
-  const result = await fetch("http://localhost:8080/");
+  const result = await fetch(`https://api.themoviedb.org/3/movie/top_rated?api_key=${apiKey}&language=en-US&page=1`);
   const resultJson = await result.json();
   const {results: movies} = resultJson;
   const movieList = document.getElementById("movie-list");
-  const imgSrc = "./assets/test.jpg";
   let listHtml = "";
 
-  movies.forEach(movie => {
+  movies.forEach(({id,poster_path: img,title,release_date}) => {
```

Comment:
Rename the variables in a camel case format while destructuring.

Reply:
Done.